int32 ra
int32 action
int32 grip
int32 vel
int32 both
---
int32 mode
